plugins/tile/tree.cpp

```diff
@@ -388,16 +410,17 @@ wf::geometry_t view_node_t::calculate_target_geometry()
     /* If view is maximized, we want to use the full available geometry */
     if (view->pending_fullscreen())
     {
-        auto vp = wset->get_current_workspace();
-        int view_vp_x = std::floor(1.0 * geometry.x / size.width);
-        int view_vp_y = std::floor(1.0 * geometry.y / size.height);
-
-        local_geometry = {
-            (view_vp_x - vp.x) * size.width,
-            (view_vp_y - vp.y) * size.height,
-            size.width,
-            size.height,
-        };
+        auto base_geometry = wset->get_attached_output()->get_relative_geometry();
```

Comment:
the base_geometry should be the size variable from above :)

issue is wset->get_attached_output() could be null in very very specific cases (i think you can only get that with some weird sequences of ipc calls)